shell/build.sh

```diff
@@ -2,6 +2,14 @@
 
 set -x
 
+  ## Download the static binaries that ansible will assume to already exist
+rm -f bin/virtctl
+wget https://github.com/kubevirt/kubevirt/releases/download/latest/virtctl-v${KUBEVIRT_VERSION}-linux-amd64 -O bin/virtctl
```

Comment:
I believe that `latest` here should be replaced by `v${KUBEVIRT_VERSION}` - judging from the URLs I see in github's release page... I did a quick test with `KUBEVIRT_VERSION=0.20.1` which failed with the URL above but worked with the change.